workflow/rocoto/tasks.py

```diff
@@ -133,7 +133,6 @@ def _get_forecast_hours(cdump, config, component='atmos') -> List[str]:
             local_config['FHMAX_HF_GFS'] = config['FHMAX_GFS']
             local_config['FHOUT_HF_GFS'] = config['FHOUT_OCNICE_GFS']
             local_config['FHOUT_GFS'] = config['FHOUT_OCNICE_GFS']
-            local_config['FHOUT'] = config['FHOUT_OCNICE']
```

Comment:
This may cause issues with GDAS. Recommend defining an `FHOUT_OCNICE` to equal `FHOUT_OCNICE_GFS` like we do with `FHMAX`.

Reply:
I defined `FHOUT_OCNICE` and set it equal to `FHOUT_OCNICE_GFS`. For GDAS cases, I set `fhout` equal to `FHOUT_OCNICE`. This should avoid any issues with GDAS.